packages/deploy/deploy/300_catalyst/300_catalyst_mint.ts

```diff
@@ -0,0 +1,42 @@
+import {DeployFunction} from 'hardhat-deploy/types';
+import {HardhatRuntimeEnvironment} from 'hardhat/types';
+
+const func: DeployFunction = async function (
+  hre: HardhatRuntimeEnvironment
+): Promise<void> {
+  const {deployments, getNamedAccounts} = hre;
+  const {execute, catchUnknownSigner, log} = deployments;
+  const {catalystMinter} = await getNamedAccounts();
+
+  // TODO Specify amounts
+  const amounts = {
+    Common: 100,
+    Uncommon: 200,
+    Rare: 300,
+    Epic: 400,
+    Legendary: 500,
+    Mythic: 600,
+  };
+  await catchUnknownSigner(
+    execute(
+      'Catalyst',
+      {from: catalystMinter, log: true},
+      'mintBatch',
+      '0x214d52880b1e4E17d020908cd8EAa988FfDD4020', // GiveawayContract
```

Comment:
I'd prefer not to hardcode this address here, we should grab the GiveawayContract address (of course check that it is the correct one) - otherwise this script will not work on testnet.